### ISSUE TITLE
Relationship [Pro] - Not documented how to change attribute showed by pivot select

### ISSUE THREAD

**Issue by realtebo** [Laravel-Backpack/docs#487]
Starts here: https://backpackforlaravel.com/docs/6.x/crud-fields#relationship-pro 

![image](https://github.com/Laravel-Backpack/docs/assets/1268699/d6aadedf-0d4b-4ea2-83b4-bda82a4b19a9)

The first [and only, often] info we need here is that if you specificy `attribute` you can switch from a select of `id`s to a select using some label of your choice

**Comment by karandatwani92** [comment]
@realtebo Thanks for pointing out the info we missed.

we will add `attribute` in the docs which let the devs choose another label. Thank you

**Comment by pxpm** [comment]
That is documented. We just need to maybe link it in more places. 
https://backpackforlaravel.com/docs/6.x/crud-fields#save-additional-data-to-pivot-table

**Comment by realtebo** [comment]
@pxpm it's not the same 

we are talking how to change the label showed in the select showed in repeatable field and in the repeatable field managed by the new relationship field.

it's not related to saving additional fata

**Comment by pxpm** [comment]
Yeah ... maybe we need to create a separate section for it. 👍 

It's there, but you need to read a bit down:
![image](https://github.com/Laravel-Backpack/docs/assets/7188159/f27b3f91-39fb-425f-a3f2-8330b338f158)

@karandatwani92 will takeover this and make this section more clear in the docs. 

Thanks for pointing it you @realtebo 🙏

**Comment by realtebo** [comment]
@pxpm Yes, i know, but my question is very specific: how can I understand that I can use 'attribute' to change the label of the select ?!  Sorry for persisting to point this specific case, but it's so common that I kindly suggest to document [or at least to back-link to a point where ALL of available attributes are documented] 

sorry, again, but 'if you want change something, change it here' it's not so fun in a documentation page.
We as developer need to know the full list of what is changeable, and what each parameters is called and what each does.

I hope to not was rude. Sorry, if I was.

**Comment by karandatwani92** [comment]
PR Submited https://github.com/Laravel-Backpack/docs/pull/496

**Comment by pxpm** [comment]
We have already changed that bit of the docs to be a full section, and added more examples. I've merged @karandatwani92 suggestion to also add the `attribute` as an example.

We will investigate the possibility of creating a page similar to [laravel helper methods list](https://laravel.com/docs/10.x/helpers#available-methods) where we list all "Field and Column attributes". 

We could create some kind of a list under each field/column docs with links to those attributes docs. 

Not sure if it wouldn't bloat the docs toooo much if we name every attribute for every field type. so we may need to "group" them by some kind of categories. 
Like using `Common Attributes`, to reference `Name, Type, Label, validationRules, hint` etc.

Alternatively we could move the docs 1 field/column per page, and in that case we think it's fine if we have a table with all supported attributes at the end. 

I will open a separate issue for that. 

Thanks for the suggestion and thanks for the PR @karandatwani92 

Cheers
